### PR TITLE
Remove the newline character that prevents using ninja

### PR DESCRIPTION
**Pull request by maxbartel**
`HASH_OUTPUT_STRIPPED` had a newline character at the end which broke the build when using Ninja. 

@tungld We tried to change the settings to enable you to click the merge button. Could you test this with this PR?

Signed-off-by: Maximilian Bartel <maximilian.bartel@amd.com>

### CODE REVIEW

**Comment by maxbartel** [comment]
@tungld Could you retrigger the tests please? I think something was wrong with jenkins.

**Comment by tungld** [comment]
> @tungld We tried to change the settings to enable you to click the merge button. Could you test this with this PR?

@maxbartel thanks, it hasn't worked yet. I still cannot update this branch with the base branch. There is no `Update Branch` botton from my view.

File: src/Version/CMakeLists.txt [hunk 1]
```diff
@@ -73,7 +73,8 @@ function(get_short_hash path out_var)
       RESULT_VARIABLE SHORT_HASH_RESULT
       OUTPUT_VARIABLE SHORT_HASH_OUTPUT
       ERROR_QUIET)
-    string(REPLACE ".git" "" SHORT_HASH_OUTPUT_STRIPPED "${SHORT_HASH_OUTPUT}")
+    string(REPLACE ".git" "" SHORT_HASH_OUTPUT_WITHOUT_GIT "${SHORT_HASH_OUTPUT}")
+    string(REPLACE "\n" "" SHORT_HASH_OUTPUT_STRIPPED "${SHORT_HASH_OUTPUT_WITHOUT_GIT}")
```
[the remark this answers is not in the document]
Comment (maxbartel) [reply src/Version/CMakeLists.txt:77]: I tried it and it didn't remove the newline character.

**Comment by maxbartel** [comment]
@tungld It seems like cross-organisation "Allow edits for maintainers" is not possible on github. See [this issue](https://github.com/orgs/community/discussions/5634). After this PR we will switch to personal forks for contributing to enable you to update the branch.

File: src/Version/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (gongsu832) [reply src/Version/CMakeLists.txt:77]: Works for me.
```linux05:/tmp# cat CMakeLists.txt
cmake_minimum_required(VERSION 3.13.4)
project(test)

execute_process(
  COMMAND echo foo
  OUTPUT_VARIABLE SHORT_HASH_OUTPUT
)
message("[${SHORT_HASH_OUTPUT}]")

execute_process(
  COMMAND echo foo
  OUTPUT_VARIABLE SHORT_HASH_OUTPUT
  OUTPUT_STRIP_TRAILING_WHITESPACE
)
message("[${SHORT_HASH_OUTPUT}]")
```
```
linux05:/tmp# cmake .
[foo
]
[foo]
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp
```

**Comment by tungld** [comment]
@jenkins-droid test this please

**Comment by tungld** [comment]
@jenkins-droid test this please

File: src/Version/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (gongsu832) [reply src/Version/CMakeLists.txt]: Indeed, I also wondered why the `--git-dir` option was used, and without actually supplying a path. I guess the intention was that the `get_short_hash` function can be called from an arbitrary directory, not necessarily the top of the repo. However, the `WORKING_DIRECTORY ${path}` option makes the `--git-dir` unnecessary.

[the remark this answers is not in the document]
Comment (maxbartel) [reply src/Version/CMakeLists.txt]: I will mark the conversation as resolved then. Could you approve the PR if you are happy with the changes, please?

**Comment by gongsu832** [comment]
@jenkins-droid test this please

**Comment by tungld** [comment]
@jenkins-droid test this please

**Pull request merged**